core/src/main/java/com/scalar/db/api/ScanBuilder.java

```diff
@@ -621,9 +591,9 @@ public static class BuildableScanOrScanAllFromExisting extends BuildableScan
           OperationBuilder.Table<BuildableScanOrScanAllFromExisting>,
```

Comment:
Since `BuildableScanOrScanAllFromExisting` is also used for changing an existing `ScanWithIndex`, it might be better `BuildableScanAllFromExisting`. But `BuildableScanOrScanAllFromExisting` is older than the cross-partition scan with filtering, so I think breaking it would be a bit aggressive. Please let me know if you have any opinions.